expansion/src/main/resources/parentPublicationFrame.json

```diff
@@ -0,0 +1,202 @@
+{
```

Comment:
Can you run a diff on the now n+ frames to see what the differences between them are? I wonder if we really need them…all :)

Reply:
Merged frame used for nvi and frame used to create expanded publication for index